sdcm/stress_thread.py

```diff
@@ -187,12 +192,16 @@ def _run_stress(self, node, loader_idx, cpu_idx, keyspace_idx):  # pylint: disab
         log_file_name = \
             os.path.join(node.logdir, f'cassandra-stress-l{loader_idx}-c{cpu_idx}-k{keyspace_idx}-{uuid.uuid4()}.log')
 
-        hdr_file_name = f"cs_hdr_{stress_cmd_opt}_{loader_idx}_c{cpu_idx}_k{keyspace_idx}_{uuid.uuid4()}.log"
+        hdr_file_name = f"cs_hdr_{stress_cmd_opt}_{loader_idx}_c{cpu_idx}_k{keyspace_idx}_{uuid.uuid4()}.hdr"
+
+        LOGGER.info("HDR log file: %s", hdr_file_name)
 
         stress_cmd = re.sub(r"%LOG%",
-                            f"-log interval 10s hdrfile={hdr_file_name}",
+                            f"-log hdrfile={hdr_file_name} interval=5s",
```

Comment:
Can you explain why you change from 10s to 5s here?

Reply:
Set as 10 seconds